pkg/report/reporter.go

```diff
@@ -171,7 +173,7 @@ func (d *defaultReporter) updateSummaryFromRecord(r Record) {
 	case StateError:
 		d.deploymentsErrorCount++
 	default:
-		panic(fmt.Sprintf("unexpected state for deployment event: %s", r.State))
+		return
```

Comment:
I also did a minor change here. We have some other record types, and using another one, not related to the summary, should not crash the application